.mergify.yml

```diff
@@ -1,13 +1,18 @@
 pull_request_rules:
   - name: Auto merge
     conditions:
-      - base=main
       - "#approved-reviews-by>=1"
       - "#changes-requested-reviews-by=0"
-      - check-success=format
-      - check-success=tests
-      - check-success=fuzz
-      - check-success=benchmark
+      - check-success=Coverage / Coverage
+      - check-success=Rust / Format
+      - check-success=Rust / Copyright Notices
+      - check-success=Rust / Checks (clippy)
+      - check-success=Rust / Checks (parachain)
+      - check-success=Rust / Checks (standalone)
+      - check-success=Rust / Quick check benchmarks
+      - check-success=Rust / Test standalone build
+      - check-success=Rust / Test parachain build
+      - check-success=Rust / Fuzz
```

Comment:
I checked the names and everything should be fine. From [here](https://github.com/zeitgeistpm/zeitgeist/blob/bac24e6f2f3bafd0f2b6baca0f57bb38b6999f2f/.github/workflows/rust.yml#L1) you use the workflow name and then you use the job names as you did.

Reply:
Except Coverage has a new workflow [here](https://github.com/zeitgeistpm/zeitgeist/blob/bac24e6f2f3bafd0f2b6baca0f57bb38b6999f2f/.github/workflows/coverage.yml#L1).